Makefile

```diff
@@ -121,15 +121,8 @@ test-in-copy:
 	make -C /lorax/ $(RUN_TESTS)
 	cp /lorax/.coverage /test-results/
 
-test-in-docker:
```

Comment:
Please don't remove this. I use it to run the tests locally.
You could change it to use podman by default though, then I could stop typing DOCKER=podman make test-in-docker :)

Reply:
What do you think @atodorov ? I could just drop this commit and just change the `DOCKER ?= docker` line in Makefile to `DOCKER ?= podman` if that's any helpful.

Reply:
Alright I did that but it seems that travis doesn't know podman... See [travis log](https://travis-ci.org/weldr/lorax/builds/610809398?utm_source=github_status&utm_medium=notification)

Reply:
Nevermind, I added the variable to .travis.yml and now it's running. Apparently there is no podman package in Ubuntu repos.